src/lib/y2network/connection_config/base.rb

```diff
@@ -38,17 +38,20 @@ class Base
       attr_accessor :interface
       # @return [BootProtocol] Bootproto
       attr_accessor :bootproto
-      # @return [IPAddr,nil]
-      attr_accessor :ip_address
-      # @return [Array<IPAddr>]
-      attr_accessor :secondary_ip_addresses
+      # @return [Array<IPConfig>]
+      attr_accessor :ip_configs
```

Comment:
maybe some helper for default would be useful? as current UI use only default, not?

Reply:
Yes, that's true. I will add it the subsequent PR so we can merge this one as soon as possible (@teclator is working on top of this moving branch).